components/Buttons/PlayButton.vue

```diff
@@ -43,32 +47,57 @@ export default Vue.extend({
       type: Array as () => BaseItemDto[],
       required: true
     },
+    disabled: {
+      type: Boolean,
+      required: false,
+      default: false
+    },
     fab: {
       type: Boolean
     },
     shuffle: {
       type: Boolean
     }
   },
+  data() {
+    return {
+      loading: false
+    };
+  },
+  computed: {
+    ...mapState('playbackManager', ['status'])
+  },
+  watch: {
+    status(): void {
+      if (this.status === PlaybackStatus.playing) {
+        this.loading = false;
+      }
```

Comment:
I don't see why this is there, as it doesn't make sense (hence why it was originally removed). Why would you disable the loading state when something is playing?

Plus, as-is, it's broken. Example: I'm loading a library page, with music in the queue, but paused. During loading, I press play. This watcher now triggers loading to be false, while the library still isn't loaded.

Reply:
Read below